### PR TITLE
Improvement/zenko 4523 lint common files

### PR DESCRIPTION
**Pull request by KazToozs**
this 0859738976080339bb576b839db3ce4b1d398487 commit related to installing a package to make the CI functional might be better in a different PR as it's not originally in the scope of this one, thoughts?

### CODE REVIEW

File: tests/ctst/common/common.ts [hunk 1]
```diff
@@ -1,59 +1,64 @@
...
+                '"ObjectLockEnabled": "Enabled",' +
+                '"Rule": {' +
+                '"DefaultRetention":' +
+                `{ "Mode": "${retentionMode}", "Days": 50 }}}`,
         });
         await S3.putObjectLockConfiguration(this.getCommandParameters());
     }
 });
 
+// eslint-disable-next-line new-cap
```
Comment (ghost) [inline tests/ctst/common/common.ts]: Same as in other PR, we can disable this globally for CTST

File: tests/ctst/common/hooks.ts [hunk 2]
```diff
@@ -4,15 +4,22 @@ import Zenko, { EntityType } from '../world/Zenko';
 // HTTPS should not cause any error for CTST
 process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0';
 
-const worlds : any = {
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const worlds : any = { // TODO: World doesn't work here becuase Zenko is missing some properties, what to do ?
     ZENKO: Zenko,
 };
```
Comment (ghost) [inline tests/ctst/common/hooks.ts]: It's expected with TS that extending a class won't work when trying to infer types compatibility. I'm not sure we can easily handle these eslint rules in this hook file, which is a bit special regarding which World to use. I would argue that it's maybe even not needed to route the right `World` as I don't see the need for any other world, as we are in Zenko...

**Review by ghost — APPROVED** [review]

**Review by welansari — APPROVED** [review]

**Comment by KazToozs** [comment]
ping

**Pull request merged**